package.json

```diff
@@ -13,17 +13,19 @@
   "scripts": {
     "export:docs": "node -r esbuild-register export/docs.ts",
     "export:overrides": "node -r esbuild-register export/overrides.ts",
+    "prettier:md": "prettier --write --parser markdown '**/*.md' --ignore-path ./LICENSE.md",
```

Comment:
This should be `prettier --check`, so it fails on problems. Not `prettier --write`, which will always pass successfully. 

(Sorry for the intermittent reviews, reviewing from phone, and still not feeling great 😕. Thank you for doing this)

Reply:
Also, is the explicit --parser flag necessary? Won't it automatically choose that? (I dunno, hence asking)

Reply:
My mistake, I thought this would just format the changes in CI before merging. 

(No worries, these things can always wait until you feel better.)